docs/SPIR-V.rst

```diff
@@ -1923,6 +1940,65 @@ Example 3: (compiled with ``-fvk-bind-globals 2 1``)
 Note that if the developer chooses to use this command line option, it is their
 responsibility to provide proper numbers and avoid binding overlaps.
 
+ResourceDescriptorHeaps & SamplerDescriptorHeaps
+------------------------------------------------
+
+The SPIR-V backend supported SM6.6 resource heaps, using 2 extensions:
+- `SPV_EXT_descriptor_indexing`
+- `VK_EXT_mutable_descriptor_type`
+
+The descriptor 0 is used to address resources in both the sampler and resource
+heaps. The binding index is the index used to subscript in the heap.
```

Comment:
I don't think that the sampler and resource descriptor heaps should have the same binding. In DirectX, samples and other resources have to be in different heaps. If someone is trying to port their DX application, this will simply cause another change they will have to make to their code.